README.md

```diff
@@ -291,6 +291,57 @@ sum by (thread_name) (idelta(tnt_cpu_thread[$__interval]))
   / scalar(idelta(tnt_cpu_total[$__interval]) / tnt_cpu_count)
 ```
 
+## Cartridge role
+
+`cartridge.roles.metrics` is role for [tarantool/cartridge](https://github.com/tarantool/cartridge). It allows to use default metrics in cartridge app and manage them with config.
+
+### Usage
+
+1. Add `metrics` package to dependencies in the .rockspec file. Ensure you're using version 0.3.0 or higher.
+```lua
+dependencies = {
+    ...
+    'metrics == 0.3.0-1',
+    ...
+}
+```
+
+2. Add `cartridge.roles.metrics` to roles list in `cartridge.cfg` in your entrypoint file (e.g. init.lua).
+```lua
+local ok, err = cartridge.cfg({
+    ...
+    roles = {
+        ...
+        'cartridge.roles.metrics',
+        ...
+    },
+})
+```
+3. After role initiation, default metrics will be enabled and global label 'alias' will be set. If you need to use any metrics package functionality, you may get it as a cartridge service and use it like a regular package after `require`:
+```lua
+local cartridge = require('cartridge')
+local metrics = cartridge.service_get('metrics')
+```
+
+4. To view metrics by API entrypoint, use configuration of the form:
+```yaml
+metrics:
+  export:
+    - path: '/path_for_json_metrics'
+      format: 'json'
+    - path: '/path_for_prometheus_metrics'
+      format: 'prometheus'
+```
+You're able to add several entrypoints of one format by different paths, like this:
```

Comment:
`It is possible to add several endpoints with the same format but different paths, for example:`